src/effects.c

```diff
@@ -470,7 +470,9 @@ void libxmp_process_fx(struct context_data *ctx, struct channel_data *xc, int ch
 		case EX_PATT_DELAY:	/* Pattern delay */
 			goto fx_patt_delay;
 		case EX_INVLOOP:	/* Invert loop / funk repeat */
-			xc->invloop.speed = fxp;
+			if (HAS_QUIRK(QUIRK_PROTRACK)) {
```

Comment:
Is this correct?

Reply:
.. especially with the lite version?

Reply:
I missed this, but there is a way for this quirk to be changed mid-playback (`xmp_set_player` with `XMP_PLAYER_MODE`) which if changed at a bad time could leave invert loop stuck on. I don't know *why* someone would do that but it's allowed by the API and probably the reason this was done the old way... ugh.

Reply:
I'll change it back to a quirk. After reviewing `QUIRK_MODRNG`, it seems safe to place `QUIRK_INVLOOP` in its position. It was the old version of `PERIOD_MODRNG` and is only still referenced in two dead loaders. Likewise, there was a `QUIRK_LINEAR` which is now `PERIOD_LINEAR`.

Reply:
OK, reverted.

Reply:
OK then. AFAICS, the playback result is identical.

Let's get this in.